app/services/load_statements.rb

```diff
@@ -24,6 +24,15 @@ def run
       %i[person_item electoral_district_item parliamentary_group_item].each do |item_attribute|
         statement.public_send("#{item_attribute}=", result[item_attribute]) if result[item_attribute].present?
       end
+
+      # Restore values from any previous Reconciliations otherwise
+      # we're reverting any manual changes or fixes that have been
+      # done:
+      %w[person party district].each do |resource_type|
+        reconciliation = statement.reconciliations.where(resource_type: resource_type).last
+        statement.public_send("#{reconciliation.send(:item_attr)}=", reconciliation.item) if reconciliation
+      end
```

Comment:
It seems a bit surprising to me to possibly set the attribute to the wrong thing in the previous loop and then correct it in this one. I think the code might be easier to read if it were one loop, which only sets the property from the upstream CSV if there's no manual reconciliation present. You lose the "self healing" aspect of the code as it is, but we could run the `restore_reconciliations` rake task to fix the existing cases that have been broken. What do you think?